src/main/metabase/api.ts

```diff
@@ -406,21 +407,81 @@ export async function getOrCreateTableExamples(tableName: string, aidColumns: st
   const dashboard = await post('api/dashboard', {
     collection_id: collectionId,
     name: `${tableMetadata.display_name} Dashboard`,
+    description: 'Dashboard with SQL query examples',
   });
   const dashboardId = dashboard.id as number;
 
   // Pin dashboard.
   await put(`/api/dashboard/${dashboardId}`, { collection_position: 1 });
 
   // Add queries to dashboard.
+
+  const layout = new CardLayout();
+  type CardData = {
```

Comment:
maybe move to `types.ts`?

Reply:
Simplified it to an anonymous type as an Array arg.